### PR TITLE
Implement quest_request_items_locale

### PR DESCRIPTION
**Pull request by AlcDenat**
**Changes proposed:** Add a new table quest_request_items_locale and code

**Target branch(es):** master

**Issues addressed:** Missing locale table

**Tests performed:** no errors, code works ingame

**Known issues and TODO list:** none

### CODE REVIEW

File: src/server/game/Globals/ObjectMgr.cpp [hunk 1]
```diff
@@ -4824,6 +4824,36 @@ void ObjectMgr::LoadQuestOfferRewardLocale()
...
+        std::string localeName = fields[1].GetString();
+
+        std::string CompletionText = fields[2].GetString();
+
+        QuestRequestItemsLocale& data = _questRequestItemsLocaleStore[id];
+        LocaleConstant locale = GetLocaleByName(localeName);
+        if (locale == LOCALE_enUS)
+            continue;
+
+        AddLocaleString(CompletionText, locale, data.RequestItemsText);
```
Comment (Shauren) [inline src/server/game/Globals/ObjectMgr.cpp]: copypaste is strong here, `CompletionText`

**Review by Shauren — CHANGES_REQUESTED** [review]

File: src/server/game/Globals/ObjectMgr.cpp (hunk 1 above)
Comment (AlcDenat) [inline src/server/game/Globals/ObjectMgr.cpp]: While 98% of this stuff is copy & paste, CompletionText is the field name in the original table and thus in the _locale table I created. I'd say it's ok to just let it be - It's never reused anyway.

Comment (joschiwald) [inline src/server/game/Globals/ObjectMgr.cpp]: `CompletionText` should be fine, rename `RequestItemsText` -> `CompletionText`

Comment (Shauren) [inline src/server/game/Globals/ObjectMgr.cpp]: just rename it to RequestItemsText to remove inconsistencies

Comment (Shauren) [inline src/server/game/Globals/ObjectMgr.cpp]: The reason I recommend keeping the `RequestItems` name is because that is the field name used in packet

Comment (AlcDenat) [inline src/server/game/Globals/ObjectMgr.cpp]: Changed it to RequestItems, should be fine now.

**Comment by joschiwald** [comment]
@Shauren i don't get your point, in packet it is named `CompletionText`

or something changed since 6.x?
```c#
public class ClientQuestGiverRequestItems
{
    public string QuestTitle                { get; set; }
    public QuestGiverRequestItems QuestData { get; set; }
    public string CompletionText            { get; set; }
}
```

**Comment by Shauren** [comment]
fine then

**Pull request closed without merging**